cli/command/command.go

```diff
@@ -144,6 +144,18 @@ func ConvertOpenShiftCommand() cli.Command {
 				Name:  "deploymentconfig,dc",
 				Usage: "Generate a OpenShift DeploymentConfig object",
 			},
+			cli.StringFlag{
```

Comment:
ah, ok i see it now ;-) this might be actually quite useful. But we need to really consider this